src/swarm/node/model/Node.d

```diff
@@ -241,6 +241,21 @@ public abstract class INodeBase : INode, INodeInfo
     }
 
 
+    /**************************************************************************
+
+        Returns:
+            the limit of the number of neo connections (i.e. the maximum number
+            of connections the node can handle in parallel) or 0 if limitation
+            is disabled
+
+     **************************************************************************/
+
+    override public size_t neo_connection_limit ( )
```

Comment:
Bah, this is useless, could we do something about this? Maybe make the `INodeInfo` abstract class and provide default implementation? I'm happy to leave it as is and wait for better days for the swarm's node framewrork.

Reply:
Puh... yeah... I wanted to mess with the horrible inheritance hierarchy as little as possible in this PR.